src/index.html

```diff
@@ -4,13 +4,28 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" data-qa="big"
+    method="get" class="bigBar">
+
+      <input type="search" class="bigBar__input"
```

Comment:
Add name attribute

Reply:
Do I have to add `name` attribute to both form and input ?

Reply:
No, only for input, I`ve selected only the input mb that is small gh bug that how 4 lines.